x/authz/keeper/grpc_query.go

```diff
@@ -132,6 +135,59 @@ func (k Keeper) GranterGrants(c context.Context, req *authz.QueryGranterGrantsRe
 	}
 
 	return &authz.QueryGranterGrantsResponse{
+		Grants:     grants,
+		Pagination: pageRes,
+	}, nil
+}
+
+// GranteeGrants implements the Query/GranteeGrants gRPC method.
+func (k Keeper) GranteeGrants(c context.Context, req *authz.QueryGranteeGrantsRequest) (*authz.QueryGranteeGrantsResponse, error) {
+	if req == nil {
+		return nil, status.Errorf(codes.InvalidArgument, "empty request")
+	}
+
+	grantee, err := sdk.AccAddressFromBech32(req.Grantee)
+	if err != nil {
+		return nil, err
+	}
+
+	ctx := sdk.UnwrapSDKContext(c)
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), GrantKey)
+
+	var authorizations []*authz.GrantAuthorization
+	pageRes, err := query.FilteredPaginate(store, req.Pagination, func(key []byte, value []byte,
+		accumulate bool) (bool, error) {
+		auth, err := unmarshalAuthorization(k.cdc, value)
+		if err != nil {
+			return false, err
+		}
+
+		granter, g := addressesFromGrantStoreKey(key)
```

Comment:
Since we are using prefix store, key won't be having that prefix included here. To fix that just append the `GrantKey` prefix here while passing it to `addressesFromGrantStoreKey`. I think this is the reason for failing tests.

Reply:
Are you sure that it still doesn't pass the entire key?